build/testing/integration.go

```diff
@@ -182,16 +227,15 @@ func importExport(ctx context.Context, base, flipt *dagger.Container, conf testC
 		namespace := conf.namespace
 		if namespace == "" {
 			namespace = "default"
+			// replace namespace in expected yaml
+			expected = strings.ReplaceAll(expected, "version: \"1.0\"\n", fmt.Sprintf("version: \"1.0\"\nnamespace: %s\n", namespace))
```

Comment:
Nice I was wondering how the `export` functionality worked as far as assertions, when data was imported with no `namespace`.

Reply:
Yeah it is a bit fiddly handling these edge cases.